src/app/log.py

```diff
@@ -26,13 +27,43 @@ def configure_logging():
                 "class": "logging.StreamHandler",
                 "formatter": "mozlog_json",
                 "stream": sys.stdout,
-            }
+            },
+            "null": {
+                "class": "logging.NullHandler",
+            },
         },
         "loggers": {
-            "request.summary": {"handlers": ["console"], "level": "INFO"},
-            "backoff": {"handlers": ["console"], "level": "INFO"},
-            "src.jbi": {"handlers": ["console"], "level": "DEBUG"},
+            "request.summary": {
+                "handlers": ["console"],
+                "level": logging.INFO,
+                "propagate": False,
+            },
+            "backoff": {
+                "handlers": ["console"],
+                "level": logging.INFO,
```

Comment:
The default logging level of `backoff` is already `INFO`, I know I'm the one that put it here, but we can remove it actually